lms/djangoapps/support/views/sso_records.py

```diff
@@ -24,5 +55,8 @@ def get(self, request, username_or_email):  # lint-amnesty, pylint: disable=miss
         except User.DoesNotExist:
             return JsonResponse([])
         user_social_auths = UserSocialAuth.objects.filter(user=user)
-        sso_records = serialize_sso_records(user_social_auths)
+        sso_records = []
+        for user_social_auth in user_social_auths:
+            user_social_auths_history = UserSocialAuth.history.filter(id=user_social_auth.id)
```

Comment:
I think this creates a 1+N queries problem, but in this context (Support Tools, and small numbers of records) I guess it doesn't matter.